### PR TITLE
fix(kafka sink): Run entire healthcheck lazily

### PR DESCRIPTION
**Pull request by LucioFranco**
Signed-off-by: Lucio Franco <luciofranco14@gmail.com>

### CODE REVIEW

File: src/sinks/kafka.rs [hunk 1]
```diff
@@ -209,9 +209,11 @@ impl Sink for KafkaSink {
 }
 
 fn healthcheck(config: KafkaSinkConfig) -> super::Healthcheck {
-    let consumer: BaseConsumer = config.to_rdkafka().unwrap().create().unwrap();
+    let client = config.to_rdkafka().unwrap();
```
Comment (lukesteensen) [inline src/sinks/kafka.rs:212]: This is technically a config, not a client, but probably not a big deal.

**Review by lukesteensen — APPROVED** [review]

File: src/sinks/kafka.rs (hunk 1 above)
Comment (LucioFranco) [reply src/sinks/kafka.rs:212]: Yeah, probably not but doesn't hurt. I think its better to just enforce that we are doing it lazily.

Comment (bruceg) [reply src/sinks/kafka.rs:212]: I think @lukesteensen was commenting on the variable name.

Comment (lukesteensen) [reply src/sinks/kafka.rs:212]: Correct :smile:

**Review by bruceg — APPROVED** [review]

File: src/sinks/kafka.rs (hunk 1 above)
Comment (LucioFranco) [reply src/sinks/kafka.rs:212]: hahah oops yeah

**Pull request merged**